lib/grape/app/initializers/pre.rb

```diff
@@ -43,12 +43,6 @@ def call(exception, *)
     ActiveRecord::Base.default_timezone = :utc
   end
 
-  if ActiveRecord.respond_to?(:legacy_connection_handling=)
-    ActiveRecord.legacy_connection_handling = false
-  else
```

Comment:
should this be an `elsif`. same here, we should support versions older than `7.1`